src/drivers/serac.cpp

```diff
@@ -113,7 +136,7 @@ int main(int argc, char* argv[])
 
   // FIXME: This and the FromInlet specialization are hacked together,
   // should be inlet["output_type"].get<OutputType>()
-  solid_solver.initializeOutput(inlet.getGlobalTable().get<serac::OutputType>(), "serac");
+  main_physics->initializeOutput(inlet.getGlobalTable().get<serac::OutputType>(), "serac");
 
   // enter the time step loop. This was modeled after example 10p.
```

Comment:
Is this comment still relevant?

Reply:
No idea. Any thoughts @joshessman-llnl ?

Reply:
I was more referring to the "This was modeled after example 10p" part.

Reply:
Heh. Whoops. And the answer is no. It should be fixed now!